### DISCUSSION TITLE
Phonemes.xml and respective flags and their values

### DISCUSSION THREAD

**Discussion by zstanecic** [RHVoice/RHVoice#450] [Q&A] [answered]
Hi to all,
Subject says it maybe all,
i want to improve the phonemes.xml for polish language. Are there somewhere more flags documented?
Where the documentation for the properties can be found?

**Comment by Olga-Yakovleva** [comment, marked as answer]
As a starting point, I used the Features and naming conventions 
Festival/Festvox used: vowel or consonant, voiced or voiceless 
consonant, vowel frontness and hight, consonant place and manner of 
articulation. Over the years I've been experimenting with adding other 
standard phonological features. You may add whichever phonological 
feature or other distinction you think can help. RHVoice will not care 
what they are and how you named them. The question generation code will 
just use those attributes to form additional phonetic questions, which 
HTS may or may not find helpful. But only a person who both knows basic 
phonetics and phonology and is familiar with the phoneme system of the 
specific natural language in question can know which features to try. 
For example, if the language has long and short vowels as separate 
phonemes, it makes sense to include the vowel length attribute.

**Comment by zstanecic** [comment]
So, if I understand you correctly, if I addthe feature central, and mark it with + or – it will try to use this info to mark that wovel is central or not?

For the reference, I am trying to improve the pronunciation of this wovel, which we marked arbitrarily as ih in our transcription.

https://en.wikipedia.org/wiki/Close-mid_central_unrounded_vowel

English has the same vowel in some situations, but we have discovered a thing with our Magda voice, that was recently published here. It reads the multiple yyyyyy, as something like y and e. In other situations, y is articulated properly.

 

Best,

Zvonimir
